source/CCustomOpcodeSystem.cpp

```diff
@@ -400,40 +400,58 @@ namespace CLEO {
 	// read string parameter according to convention on strings
 	char *readString(CRunningScript *thread, char* buf = nullptr, BYTE size = 0)
 	{
+		if (size == 0) size = 0xFF; // max

```

Comment:
0xFF probably should be defined as a const MAX_STR_LEN

Reply:
Right! Updated.